cloudbuild/nightly.yaml

```diff
@@ -0,0 +1,39 @@
+steps:
+# 1. Create a Docker image containing flink-bigquery-connector repo
+  - name: 'gcr.io/cloud-builders/docker'
+    id: 'docker-build'
+    args: ['build', '--tag=gcr.io/$PROJECT_ID/dataproc-flink-bigquery-connector-nightly', '-f', 'cloudbuild/Dockerfile', '.']
+
+# 2. Fetch maven and dependencies
+  - name: 'gcr.io/$PROJECT_ID/dataproc-flink-bigquery-connector-nightly'
+    id: 'init'
+    waitFor: ['docker-build']
+    entrypoint: 'bash'
+    args: ['/workspace/cloudbuild/nightly.sh', 'init']
+    env:
+      - 'GCS_JAR_LOCATION=${_GCS_JAR_LOCATION}'
+      - 'MVN_JAR_LOCATION=${_MVN_JAR_LOCATION}'
+   
+# 3. Start the simple e2e tests
+  - name: 'gcr.io/$PROJECT_ID/dataproc-flink-bigquery-connector-nightly'
+    id: 'e2e-bounded-read-tests'
```

Comment:
We're running e2e tests after copying the jar to GCS. Is this GCS location going to be public?
If yes, we should instead consider copying it to a staging location first, and make it public in a separate step after the e2e tests pass.

Reply:
No, this is not a public bucket.